src/ot_orchestration/operators/gce.py

```diff
@@ -0,0 +1,733 @@
+    :param zone: The zone where the instance will be created (default is f'{GCP_REGION}-b').
+    :param instance_name: Name of the instance name that will run the workload.
+    :param labels: Optional dict of labels to apply to the instance on top of the default ones, which
+        are `team: open-targets`, `product: platform`, `environment: development` or `production`,
+        and `created_by: unified-orchestrator`. Refer to `controlled vocabularies
+        <https://github.com/opentargets/controlled-vocabularies/blob/main/infrastructure.yaml>`__ for
+        more information.
+    :param container_image: Container image to run.
+    :param container_command: Command to run inside the container (optional).
+    :param container_args: Arguments to pass to the container (optional).
+    :param container_env: Environment variables to pass to the container (optional).
+    :param container_service_account: Service account to use for the instance (default default).
+    :param container_scopes: A list of extra scopes to add to the service account if any are needed.
+    :param container_files: Files to copy to the instance (optional). This is a dictionary where
+        the key is a GCS path in the form `gs://bucket/path/to/file` and the value is the path
+        where the file will be copied to in the instance. This is intented for small files needed
+        to run the workload (like the configuration). Large files should be downloaded by the
+        workload itself. The paths specified as values will be relative to the `/home/app` directory
+        and will be created if they don't exist. Inside the docker container, they will be mounted
+        under the root directory.
+    :param machine_type: Machine type to use for the instance (default e2-standard-2).
+    :param work_disk_size_gb: If present, a second disk with the specified size in GB will be
+        attached to the instance besides the boot disk, to be used by the workload. The disk will
+        be formatted with ext4 and mounted under `/mnt/disks/work`. The instance will have write
+        permissions to the disk. The disk will be deleted when the instance is deleted.
+    :param gcp_conn_id: The connection ID to use connecting to Google Cloud.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :param deferrable: If True, run the sensor in deferrable mode.
```

Comment:
If it's only implemented for defferable mode, raise NotImplementedError when defferable is set to False